conformance/utils/suite/suite.go

```diff
@@ -152,11 +160,16 @@ type ConformanceTest struct {
 	Slow        bool
 	Parallel    bool
 	Test        func(*testing.T, *ConformanceTestSuite)
+	MinChannel  string
 }
 
 // Run runs an individual tests, applying and cleaning up the required manifests
 // before calling the Test function.
 func (test *ConformanceTest) Run(t *testing.T, suite *ConformanceTestSuite) {
+	const (
+		Experimental Channel = 1
+		Standard     Channel = 2
+	)
```

Comment:
I think we should define this alongside the "SupportedFeature" and "ExemptFeature" types at the top of this file. Maybe call the new type "GatewayChannel"?

Reply:
Okay, Rob! I'll fix it by your sugestion. Thanks for guiding me.